lib/cli/options.js

```diff
@@ -195,6 +195,11 @@ const loadPkgRc = (args = {}) => {
           `Unable to read/parse ${filepath}: ${err}`,
           filepath
         );
+      } else if (err.toString().includes('SyntaxError')) {
+        throw createUnparsableFileError(
```

Comment:
(On my phone) Will throwing here result in a warning? Feels more like an error to me?

Reply:
It will fail with an error, these exceptions are not caught (we could, possibly, catch them at the top level of the CLI and exit cleanly in order to avoid barfing stack traces on the user...)